google/cloud/pubsub/internal/tracing_message_batch_test.cc

```diff
@@ -179,14 +180,15 @@ TEST(TracingMessageBatch, FlushSmallBatch) {
                                  "messaging.pubsub.message.link", 1)))))));
 }
 
-// TODO(#12528): Update test when impl is added
-TEST(TracingMessageBatch, FlushLargeBatch) {
+TEST(TracingMessageBatch, FlushBatchWithOtelLimit) {
+  int const k_batch_size = 128;
```

Comment:
Looks clang-tidy converted `kBatchSize` to `k_batch_size`.... 

```cc
auto constexpr kBatchSize = 128;
```

will work.